mcs/class/System.ServiceModel.Web/Assembly/AssemblyInfo.cs

```diff
@@ -55,10 +55,8 @@
 [assembly: AssemblyDelaySign (true)]
 #if MOBILE
 [assembly: AssemblyInformationalVersion (Consts.FxFileVersion)]
-[assembly: AssemblyKeyFile ("../silverlight.pub")]
 #else
 [assembly: AssemblyInformationalVersion ("3.5.594.0")]
```

Comment:
I don't know of a good reason to have this distinction on InformationalVersion, @marek-safar do you?